_std/types.dm

```diff
@@ -233,6 +233,7 @@ var/list/list/by_cat = list()
 
 /typeinfo/atom
 	parent_type = /typeinfo/datum
+	var/random_subtypes = null
```

Comment:
Add code doc; it's unclear what `random_subtypes` actually means and how it should be used.